docs/ocm/modelmapping/oci/README.md

```diff
@@ -48,6 +48,18 @@ access specification.
 
 </div>
 
+## Version Mapping
+
+The Open Component Model supports version names according to [semantic versioning](https://semver.org/).
```

Comment:
nit: since we allow `v` prefix, we are not strictly semver-compliant

Reply:
added a footnote